src/pages/Register.ts

```diff
@@ -91,8 +91,9 @@ const Register = () => {
         ...orgFields,
       });
       authStore.update((state) => ({ ...state, registrationStatus: 'registeredWithoutTos' }));
-      await refreshTerraProfile();
-      Ajax().Metrics.captureEvent(Events.user.register);
+      await refreshTerraProfile().then(() => {
+        Ajax().Metrics.captureEvent(Events.user.register);
+      });
```

Comment:
Suggest changing this back. It does the same thing, but mixing `async/await` and Promise methods makes it less clear.